### ISSUE TITLE
if options.dir has trailing slash, error is thrown 

### ISSUE THREAD

**Issue by auroraisluna** [auroraisluna/fastify-router#2]
Using the example: 
```js
import { loadRoutes } from 'fastifyrouter.js';

import Fastify from 'fastify';
const fastify = Fastify();

await loadRoutes(fastify, { dir: './routes', log: true, method: 'GET' });```
```

Module will work. However, if dir is set to `./routes/` instead you get this error: 

```fix
AssertionError [ERR_ASSERTION]: The first character of a path should be `/` or `*`
```

**Comment by auroraisluna** [comment]
Potential fixes: 
1. Remove trailing slashes from paths and continue with life [does MacOS/Linux support this?]
2. Throw error (a better one than the current one) about this issue

**Comment by auroraisluna** [comment]
Fixed in v1.0.3
